include/fastrtps/qos/ParameterTypes.h

```diff
@@ -124,31 +124,48 @@ class Parameter_t
 {
 public:
 
-    RTPS_DllAPI Parameter_t() : Pid(PID_PAD), length(0) { }
+    RTPS_DllAPI Parameter_t()
+        : Pid(PID_PAD)
+        , length(0)
```

Comment:
**In-class initializers** were introduced exactly for that. But I'm an oldtimer and keep searching for initialization values in the constructor (where all are bundled together) instead of moving around the class declaration looking for values 🙈

Reply:
Yes, but after some days you check the headers, the advantage of the in-class initializers is that they greatly simplify a lot of basic code, most classes will get default constructors, especially all IDL generated code can be greatly reduced. See https://github.com/RemedyIT/idl2cpp11 for what we generate for the IDL to C++11 language mapping